src/library/ownedTraitIDs.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Represents owned traits and IDs extracted from NFT metadata.
+ */
+export class OwnedTraitIDs {
```

Comment:
Could we rename this to something like `OwnedNFTTraitsIds` ? just to clarify we're talking about NFTs